geolocator_platform_interface/lib/src/geolocator_platform_interface.dart

```diff
@@ -159,12 +159,14 @@ abstract class GeolocatorPlatform extends PlatformInterface {
     throw UnimplementedError('getPositionStream() has not been implemented.');
   }
 
-  /// Returns a [Future] containing a [LocationAccuracyStatus].
+  /// Returns a [Future] containing a [LocationAccuracyStatus] (iOS only).
   ///
   /// When on iOS the user has given permission for approximate location,
   /// [LocationAccuracyStatus.reduced] will be returned, if the user gave
   /// permission for precise/full accuracy location, [LocationAccuracyStatus.precise]
   /// will be returned.
+  /// When executing the method on Android device's,
+  /// [LocationAccuracyStatus.unknown] will be returned.
```

Comment:
The geolocator also supports the `Web` platform and hopefully soon also others like Linux, MacOS, Windows and embedded systems. So this documentation is not correct maybe update it to:

 ```suggestion
  /// When executing the method on platforms that don't support location 
  /// accuracy features [LocationAccuracyStatus.unknown] should be returned.
```